### ISSUE TITLE
Why do some pages don't have titles in the deployed site?

### ISSUE THREAD

**Issue by sivaraam** [git/git.github.io#489]
Consider the following pages in https://git.github.io/ :

- https://git.github.io/Hacking-Git/
- https://git.github.io/General-Microproject-Information/

Observe that even though they have a proper title mentioned in the corresponding Markdown files, the titles don't appear in the body of the text. It is only visible in the link and the title of the tab in the browser. Is there a particular reason for this? Or is there no particular reason for this?

**Comment by chriscool** [comment]
It looks like the titles that are mentioned in the Markdown files are only in the header parts of the files, and I guess it's part of how these static pages work that a title should not just be in the header if it should appear on the page itself.

**Comment by sivaraam** [comment]
I just realized that I never mentioned why I was wondering about this. The reason is that I had a tab from git.github.io open. When I returned to it after a while, I found it hard to identify what page it was as there was no title. That's when I started wondering why the page title is present only in the header.

> 
> 
> It looks like the titles that are mentioned in the Markdown files are only in the header parts of the files, and I guess it's part of how these static pages work that a title should not just be in the header if it should appear on the page itself.

Right. I certainly believe there would be a way to make the title in the header appear in the page itself without having to include it manually in all the pages. Actually, I just briefly looked into it and found a way to do that. So, would a change to include the title in the page itself be welcome? I think it might require some special handling for Rev News as all rev news articles already have a header.

**Comment by chriscool** [comment]
Yeah, I think it could help people avoid mistakes and identify rapidly which page they are on. So I am ok with such a change.
